src/components/Card/src/CardActions/bem-mapping.jsx

```diff
@@ -0,0 +1,3 @@
+export const cardActionClasses = {
+    root: 'denhaag-card-actions'
```

Comment:
I'm not 100% sure, but I think according to BEM, actions is an element of the denhaag-card. So it would be `denhaag-card__actions`

Reply:
Yea I agree, as this subcomponent is always and only used as part of a Card this would be an element in the card block.